apps/backend/src/handlers/getAllForms.ts

```diff
@@ -7,6 +7,18 @@ import { validateMethodType } from '../utils/validateMethodType.js';
  * An HTTP get method to get all forms from the QLDB table.
  */
 export const getAllFormsHandler = async (event: APIGatewayEvent) => {
+  const headers = {
+    'Access-Control-Allow-Headers': 'Content-Type, Access-Control-Allow-Origin, Authorization',
+    'Access-Control-Allow-Origin': '*',
+    'Access-Control-Allow-Methods': 'GET',
+  };
+
+  if (event.httpMethod !== 'GET') {
+    return {
+      statusCode: 400,
+      headers,
+      body: `getMethod only accepts GET method, you tried: ${event.httpMethod} method.`,
+    };
```

Comment:
wait sorry I don't think you merged main in properly, we're missing a closing brace here